servers/audio/audio_stream.h

```diff
@@ -71,6 +71,8 @@ class AudioStreamPlaybackResampled : public AudioStreamPlayback {
 	virtual void _mix_internal(AudioFrame *p_buffer, int p_frames) = 0;
 	virtual float get_stream_sampling_rate() = 0;
 
+	bool ready = false;
```

Comment:
I'd move this further up in the `private` section with other members variables (i.e. after `mix_offset`).

Reply:
Fixed!